File: telemetry/scripts/track-ipfs-net.py
import psutil

# Get the process ID of the binary you want to monitor
pid = 56304

# Get the network I/O counters for the process
io_counters = psutil.net_io_counters(pid)
print(io_counters)

# Print the number of bytes sent and received by the process
print(f"Bytes sent: {io_counters.bytes_sent}")
print(f"Bytes received: {io_counters.bytes_recv}")

File: telemetry/scripts/track-ipfs-net.js
import psList from 'ps-list'
// Get the process ID of the binary you want to monitor
const pid = 56304;

// Get the network I/O statistics for the process
psList().then(processes => {
    const process = processes.find(p => p.pid === pid);
    console.log(process)
    if (process && process.netio) {
        console.log(`Bytes sent: ${process.netio.bytesSent}`);
        console.log(`Bytes received: ${process.netio.bytesRecv}`);
    }
});